python/date.py

```diff
@@ -3,6 +3,14 @@
 import datetime
 
 from mage.date.constants import Conversion, Epoch
+from mage.date.unit_conversion import to_int, to_timedelta
+
+
+MILLISECOND = {"ms", "milli", "millis", "milliseconds"}
+SECOND = {"s", "second", "seconds"}
+MINUTE = {"m", "minute", "minutes"}
+HOUR = {"h", "hour", "hours"}
+DAY = {"d", "day", "days"}
```

Comment:
This is unused, otherwise can be imported from mage.date.constants if needed

Reply:
This was a duplicate, thanks for pointing it out!